### ISSUE TITLE
Escaping UserStrings for svg representation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [luisvt/oryx-editor#116]
```
What steps will reproduce the problem?
1. Set Title of Activity like "... & ..."
2. Save
3. Load the svg representation

What is the expected output?
The svg image

What do you see instead?
parse error, because '&' was not escaped 



```

Original issue reported on code.google.com by `ole.ecke...@gmail.com` on 19 May 2008 at 8:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `NicoPete...@gmail.com` on 23 May 2008 at 3:38
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `NicoPete...@gmail.com` on 23 May 2008 at 3:38
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in issue #445
```

Original comment by `NicoPete...@gmail.com` on 3 Jun 2008 at 3:59
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `NicoPete...@gmail.com` on 11 Jun 2008 at 12:32
- Changed state: **Verified**
